cpack/debian/postinst

```diff
@@ -0,0 +1,35 @@
+#!/bin/sh
+
+set -e
+
+case "${1}" in
+	configure)
+		chmod +x /etc/init.d/fluent-bit
+		;;
+
+	abort-upgrade|abort-remove|abort-deconfigure)
+
+		;;
+
+	*)
+		echo "postinst called with unknown argument \`${1}'" >&2
+		exit 1
+		;;
+esac
+
+# Automatically added by dh_installinit/12.10
```

Comment:
These are copied from " Automatically added by dh_installinit/12.10" and they are not getting added automatically now, having it here will cause confusion for users seeing it in their `postinst`, `postrm` and so on.